review.rake

```diff
@@ -4,7 +4,7 @@ require 'json'
 CONFIG = JSON.parse(ENV['CONFIG'])
 SLACK_TOKEN    = ENV['SLACK_TOKEN']
 SLACK_BOT_NAME = 'reviewbot'
-SLACK_BOT_ICON = ':smile_cat:'
+SLACK_BOT_ICON = ':robot_face:'
```

Comment:
I really like what you've done here. But I absolutely cannot have this go to production. We need a full basecamp discussion about the correct emoji for this bot

Reply:
![](http://i0.kym-cdn.com/entries/icons/mobile/000/006/026/NOTSUREIF.jpg)